projects/packages/sync/src/modules/class-posts.php

```diff
@@ -896,7 +896,7 @@ public function filter_posts_and_metadata_max_size( $posts, $metadata ) {
 			}
 			// Always allow the first post with its metadata.
 			if ( empty( $filtered_post_ids ) || ( $current_size + $post_content_size + $metadata_size ) <= ( self::MAX_SIZE_FULL_SYNC ) ) {
-				$filtered_post_ids[] = $post->ID;
+				$filtered_post_ids[] = strval( $post->ID );
```

Comment:
This is to make sure that `$filtered_post_ids` will have identical structure than what is coming from DB. This will help later when using `$status['last_sent']` to compare.

Reply:
Why not add the `strval` part only when comparing?